win32ss/user/ntuser/nonclient.c

```diff
@@ -263,7 +263,7 @@ DefWndDoSizeMove(PWND pwnd, WORD wParam)
    ExStyle = pwnd->ExStyle;
    iconic = (Style & WS_MINIMIZE) != 0;
 
-   if ((Style & WS_MAXIMIZE) || !IntIsWindowVisible(pwnd)) return;
+   if (!IntIsWindowVisible(pwnd)) return;
```

Comment:
Are you sure about this change?

Reply:
I thought about MDI windows, but they have no caption bar when maximized. Other side-effects not appeared

Reply:
@HBelusca revert this change if you have some doubts?

Reply:
@yagoulas  hi, can you check this place please? may be you have idea what side effect can come up after removing this WS_MAXIMIZE-check

Reply:
OK I've done some investigations, using WinSpy on Windows 2003. It's possible there to move a window that has the `WS_MAXIMIZE` style "programmatically". What is interesting to observe however, is that this window won't be movable using the mouse as long as its **size** >= its original maximized size -- even if this window has been programmatically moved to some other position. Only when its **size** is made smaller (programmatically) than this original maximized size, could the "maximized" window be moved again with the mouse. If, after this step, the window is made again larger (or equal) to its original maximized size, then it cannot be moved with the mouse again.

Reply:
returned this check with exception for SC_MOVE command, so if somehow SC_SIZE come in it will behave same as it going now